### DISCUSSION TITLE
Timestamp of the transcript

### DISCUSSION THREAD

**Discussion by SalvatoreRa** [openai/whisper#720] [Q&A]
Hi,

Thank you for this amazing model. I would like it if it is possible to have the timestamp of the transcript.

For example, at the time of the video, a certain sentence is said in the audio. just a simple example:

Taking a video x, when in the transcript there is the sentence y, extracting also at what time in the video is said, like:

00.00.10 Hi everyone, welcome here...

00.10.05 We will talk about...

Thank you very much

**Comment by iburunat** [comment]
You could use the the `write_srt` module: `from whisper.utils import write_srt`.

**Reply by orangeduu** [reply]
Do you have an example of how that would look in python? For the following lines of code, where would I input this?
`model = whisper.load_model("small")
result = model.transcribe("spanishtest.mp3", language='spanish', task='translate', fp16=False)
print(result["text"])`
